communications/newsletters/newsletter_11_Feb2020.md

```diff
@@ -0,0 +1,198 @@
+Read about them and more details on other topics below 👇
+
+## Community meetings
+
+### Collaborate Cafes
+
+Join us at the next collaboration cafe on 19 February at 19:00 GMT ([your time zone](https://arewemeetingyet.com/London/2020-02-19/19:00/TuringWay-CollaborationCafe)).
+Sign up on [this HackMD](https://hackmd.io/@KirstieJane/CollabCafe).
+
+These online meetings are held using the [Pomodoro Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique) to co-work without distractions to add your contributions to *The Turing Way* project.
+These contributions can be from fixing bugs to reviewing existing chapters or adding your own content to the book.
+*The Turing Way* team members will be present to help you troubleshoot and answer your questions.
+
+Thanks to the positive response from our past attendees, we will continue to hold these working hours on the first Wednesday at 15:00 GMT and third Wednesday at 19:00 GMT of each month.
+You can see the exact date and time for the next few sessions on [this GitHub page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md).
+
+## News from the community
+
+### Congratulations to our 15 selected book dash participants!
+
+We are excited to announce that 15 members have been invited to attend the upcoming book dash hosted at The Alan Turing Institute, London from 20 to 21 February 2020.
+These members are (listed alphabetically): [Adina Wagner](https://twitter.com/AdinaKrik), [Alex Chan](https://twitter.com/alexwlchan), [Anna Hadjitofi](https://www.turing.ac.uk/people/researchers/anna-hadjitofi), [Arielle Bennett](https://uk.linkedin.com/in/ariellebennett), [Barbara Vreede](https://twitter.com/barbaravreede), [Christina Hitrova](https://twitter.com/C_Hitrova), [Esther Plomp](https://twitter.com/PhDToothFAIRy), [Frances Madden](https://twitter.com/maddenfc), [Heidi Seibold](https://twitter.com/HeidiBaya), [Kesson Magid](https://twitter.com/kesson), [Martina G. Vilas](https://twitter.com/martinagvilas), [Mateusz Kuzak](https://twitter.com/matkuzak), [Nathan Begbie](https://www.linkedin.com/in/nathan-begbie-161902173/), [Nicolás Alessandroni](https://twitter.com/nalessandroni), and [Tony Yang](https://twitter.com/tony_yzy) 💖 🙌 🚀 🌟
+
+One of the past attendees, [Jade Pickering](https://twitter.com/Jade_Pickering) will be returning as a helper along with the core contributors [Louise Bowler](https://www.turing.ac.uk/people/researchers/louise-bowler) and [Sarah Gibson](https://twitter.com/drsarahlgibson).
+Those who are not located in London have also received travel and accommodation support from *The Turing Way* project.
+We are looking forward to welcoming them and working with them in person!
+
+#### Your comments are needed on the rubric for selection
+
+A call for application was opened in December 2019, inviting applications from the interested candidates sharing their ideas for contribution to *The Turing Way* project.
```

Comment:
```suggestion
An open call for applications was opened in December 2019 for the book dash, inviting ideas from the interested candidates for their contributions to *The Turing Way* project.
```

Reply:
As a separate (independent paragraph), I had not clarified that we are still talking about the book dash.